source/about/solutions.rst

```diff
@@ -0,0 +1,16 @@
+Use cases
```

Comment:
Since this is a brand new page, let's rename the file (and page URL) for improved SEO